scenes/game_elements/props/hint/input_key/directional_input_hint.gd

```diff
@@ -100,29 +97,34 @@ func _physics_process(_delta: float) -> void:
 			match current_device:
 				InputHelper.DEVICE_XBOX_CONTROLLER:
 					texture = (
-						xbox_pressed_texture if xbox_pressed_texture else controller_pressed_texture
+						xbox_pressed_texture if xbox_pressed_texture else null
 					)
```

Comment:
```suggestion
					texture = xbox_pressed_texture
```

because `x if x else null`, when `x` is either `null` or a non-`null` reference, means the same as `x`.

Same change on the other branches.